crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -330,6 +375,74 @@ impl CumulocityConverter {
         Ok(messages)
     }
 
+    async fn new_try_convert_event(
```

Comment:
The same comment as above for measurements. What's the difference from the original function?